contracts/sol6/KyberNetworkProxy.sol

```diff
@@ -375,7 +375,11 @@ contract KyberNetworkProxy is
         uint256 srcAmount,
         address destAddress
     ) internal returns (UserBalance memory balanceBefore) {
-        require(src == ETH_TOKEN_ADDRESS || msg.value == 0, "msg.value should be 0");
+        if (src == ETH_TOKEN_ADDRESS) {
+            require(msg.value == srcAmount, "Sent eth not equal to srcAmount");
```

Comment:
look inconsistent revert messages, we don't use uppercase for first letter :/

Reply:
right
thanks